linera-storage-service/src/child.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use linera_service::util::CommandExt;
+use std::time::Duration;
+use tokio::process::{Child, Command};
+
+/// Configuration for a storage service running as a child process
+pub struct StorageServiceChild {
+    endpoint: String,
+    binary: String,
+}
+
+/// The tests being done have to run in parallel.
+/// For that we spanned a storage-service per test.
+/// In order to avoid collision, we use a different port per test.
+///
```

Comment:
This comment doesn't belong here (should be probably be a `//`-comment next to tests). Anyway, it is also not very useful, so I would remove it.

Reply:
Ok.